app/assets/javascripts/components/toolbar-dropdown.js

```diff
@@ -32,15 +37,31 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
   }
 
-  ToolbarDropdown.prototype.handleClick = function (event) {
+  ToolbarDropdown.prototype.handleTitleClick = function (event) {
+    this.toggleDropdownContainer()
+  }
+
+  ToolbarDropdown.prototype.handleListItemClick = function (event) {
     this.closeToolbarDropdown()
   }
 
   ToolbarDropdown.prototype.closeToolbarDropdown = function (event) {
-    if (this.$module.hasAttribute('open')) {
-      this.$title.click()
+    this.$module.removeAttribute('open')
+  }
+
+  ToolbarDropdown.prototype.toggleDropdownContainer = function (event) {
+    if (!this.$module.hasAttribute('open')) {
+      this.$module.setAttribute('open', 'open')
+    } else {
+      this.$module.removeAttribute('open')
     }
   }
 
+  ToolbarDropdown.prototype.hasDetailsSupport = function () {
+    var el = document.createElement('DETAILS')
```

Comment:
I wonder if this is always returning false? Looking at the docs for the element, I can't see why it would contain an `open` attribute immediately after creation. Also, should we be cleaning it up?

Reply:
As an afterthought, I'm wondering if this could be separated as a 'true' polyfill? I can see it's tied in to the behaviour here, so the motivation is mainly to simplify the change.

Reply:
Happy to separate this into it's own PR to give more room for explanation and code documentation.

Reply:
Sorry, I didn't mean to evade your question but to make things simpler. Assuming you're asking about the result of `GOVUK.Modules.ToolbarDropdown.prototype.hasDetailsSupport()` (not the one of `document.createElement('DETAILS')` where the comment was added) the result is `true` for browsers with support and `false` otherwise. Tested in all the browsers we support (including IE11 where it returns false).

Reply:
@alex-ju thanks for stopping by to remind me how JS works! To answer my own question: `createElement` doesn't attach things to the DOM, so the object will get cleaned up.

Reply:
As discussed, we think we should add a comment here to explain the unintuitive behaviour of `createElement` and how it auto-populates attributes on whatever it returns. This should clarify whether or not we expect to see the (nil valued) `open` attribute.